lib/python/qmk/cli/new/keymap.py

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env python
```

Comment:
I rewrote the new_keymap.py file but scoped out setting logging level for this first draft. If you all believe the we can scope in setting the logging level for this file, see the todos (line 18) for the debug code